### DISCUSSION TITLE
I need help getting a background image to work in my build/deployment: React, Vite, Tailwind.css

### DISCUSSION THREAD

**Discussion by jesseBlank** [community/community#125403] [Programming Help] [answered]
### Select Topic Area

General

### Body

Hello,

My background image is not displaying on my deployed site. However, it is working on my local host. I have seen posts like this that have similar issues but none of those solutions are working for me.

I have noticed the image is not being being moved to my dist folder. When I manually move it and attempt to deploy, it is automatically removed from the dist folder.

this is my css code:
.bg-image {
    background-image: URL(src/assets/desertSky.jpeg);
}

my repo: https://github.com/jesseBlank/portfolio
my site: https://jesseblank.github.io/portfolio/

Any help would be huge!

Thanks.

**Comment by LiteBrite82** [comment]
Hi @jesseBlank 

I forked your repo and messed around with your code a bit. I dont have a solution for you yet - I'm not familiar with tailwind. Im not sure how to mix custom CSS with Tailwind - I use Material UI.

I can tell you its not a pages issue; it is a coding and build issue (configuration of tailwind, import in index.html, file naming, file referencing). 

To start, take a look at your configuration for tailwind compared to the [documentation](https://tailwindcss.com/docs/guides/vite). It looks like you may have some adjusting to do. 

Then, hopefully, someone in the community can help with the file reference issues. I feel like its something super simple; buried in documentation. 

I'm going to move this over to Programming Help for more focused visibility.

**Reply by LiteBrite82** [reply]
Also, consider editing the discussion title to include tailwind.css and Vite. Ex: "_I need help getting a background image to work in my build/deployment: React, Vite, Tailwind.css_"

**Reply by jesseBlank** [reply]
Okay. Thank you for your help.

**Reply by LiteBrite82** [reply, marked as answer]
I found a good [YouTube walkthrough](https://www.youtube.com/watch?v=P9iPATix0u0) by @pixegami that covers a good way to do it in great detail - skip to 26:30 

Thanks @pixegami !

**Reply by jesseBlank** [reply]
Perfect. That's it! Thank you so much for your help!

**Reply by LiteBrite82** [reply]
That looks Awesome!
